pkg/cmsis-dsp/Makefile

```diff
@@ -5,6 +5,9 @@ PKG_LICENSE=Apache-2.0
 
 include $(RIOTBASE)/pkg/pkg.mk
 
+# include makefiles utils tools
+include $(RIOTMAKE)/utils/strings.mk
```

Comment:
Isn't this available early on?

Reply:
It was returning empty string, so I'm guessing it's not.